### ISSUE TITLE
hot potato doesn't show up in hand for other clients

### ISSUE THREAD

**Issue by Slava0135** [space-wizards/space-station-14#16369]
## Description

**Reproduction**

**Screenshots**

**Additional context**

**Comment by Slava0135** [comment]
i think its better to disable it for now

**Comment by AJCM-git** [comment]
just remove it from the catalog for now

**Comment by deltanedas** [comment]
you can still see the particles no

**Comment by Slava0135** [comment]
> you can still see the particles no

yes but you need to attempt to drop it or something to make it show in hand, and you can't pass it before that

**Comment by kontakt** [comment]
Resolved.
